src/grid/onedgrid.py

```diff
@@ -33,16 +33,24 @@ def generate_onedgrid(npoints, *args):
 
 
 def GaussLaguerre(npoints, alpha=0):
-    r"""Generate Gauss-Laguerre grid.
+    r"""Generate a grid based on generalized Gauss-Laguerre grid.
 
+    Generalizeed Gauss Laguerre grid is defined as:
     .. math::
-        \int_{0}^{\infty} e^{-x} f(x)dx \approx \sum_{i=1}^n w_i f(x_i)
+        $$\int_{0}^{\infty} x^{\alpha}e^{-x} f(x)dx \approx
+        \sum_{i=1}^n w_i f(x_i)
+
+    This integration grid is defined as :
+    .. math::
+        $$\int_{0}^{\infty} f(x)dx \approx
```

Comment:
Sphinx use `.. math::` as math directive. So it would be better to remove the `$$` sign.